bitnami/concourse/templates/NOTES.txt

```diff
@@ -0,0 +1,29 @@
+
+* Concourse can be accessed:
+
+  * Within your cluster, at the following DNS name at port {{ .Values.web.bindPort }}:
```

Comment:
I see that you took part of the notes.txt from th upstream solution. It seems that we can re-use the NOTES.txt message from other web applications to add more consisteny. Also, does this application have credentials? I would add a note for getting them with the proper command.